src/pages/workspace/accounting/qbo/export/QuickbooksCompanyCardExpenseAccountPage.tsx

```diff
@@ -78,9 +78,15 @@ function QuickbooksCompanyCardExpenseAccountPage({policy}: WithPolicyConnections
                     )}
                     <OfflineWithFeedback pendingAction={pendingFields?.nonReimbursableBillDefaultVendor}>
                         <MenuItemWithTopDescription
-                            title={nonReimbursableBillDefaultVendorObject?.name}
+                            title={isVendorSelected ? nonReimbursableBillDefaultVendorObject?.name : nonReimbursableExpensesAccount?.name}
                             description={isVendorSelected ? translate('workspace.qbo.vendor') : translate('workspace.qbo.account')}
-                            onPress={() => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKS_ONLINE_NON_REIMBURSABLE_DEFAULT_VENDOR_SELECT.getRoute(policyID))}
+                            onPress={() =>
+                                Navigation.navigate(
+                                    isVendorSelected
+                                        ? ROUTES.POLICY_ACCOUNTING_QUICKBOOKS_ONLINE_NON_REIMBURSABLE_DEFAULT_VENDOR_SELECT.getRoute(policyID)
+                                        : ROUTES.POLICY_ACCOUNTING_QUICKBOOKS_ONLINE_COMPANY_CARD_EXPENSE_ACCOUNT_SELECT.getRoute(policyID),
+                                )
+                            }
                             brickRoadIndicator={errorFields?.nonReimbursableBillDefaultVendor ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
                             shouldShowRightIcon
                             error={errorFields?.nonReimbursableBillDefaultVendor ? translate('common.genericErrorMessage') : undefined}
```

Comment:
Update pendingAction, brickRoadIndicator and error

Reply:
Actually this is looking too different that we should render another menu item

Reply:
@s77rt fixed here: https://github.com/Expensify/App/pull/42102/commits/27b8e9cd54614024370236afd07f2c07e01a8211